pages/maps/_id/edit.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class=" justify-center items-center flex flex-grow">
+    <div class="bg-blue-900 p-5 rounded-sm main-container">
+      <form @submit.prevent="onSubmit" class="flex flex-col">
+        <h1 class="text-white text-xl mb-2">Edit Map</h1>
+        <!-- <p v-if="errors.length" class="text-red-300 bg-red-800 p-3 mb-3 rounded-sm">
+          <strong>Please correct the following error(s):</strong>
+          <ul>
+            <li v-for="error in errors">{{ error }}</li>
+          </ul>
+        </p> -->
+        <input
+          type="text"
+          name="name"
+          v-model="name"
+          class="mb-4 p-2 rounded-sm"
```

Comment:
I use these personally, https://tailwindcss-custom-forms.netlify.app/

made a bunch of my own custom ones, light/dark theme